packages/react-magma-dom/src/components/Tabs/TabsScrollSpyContainer.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useCallback } from 'react';
+import styled from '@emotion/styled';
+import { ScrollSpy } from './utils';
+import { Tabs, TabsOrientation } from './Tabs';
+import { Tab } from './Tab';
+import { TabsContainer } from './TabsContainer';
+import { toCamelCase } from '../../utils';
+
+export interface TabScrollSpyContainerProps
+  extends React.HTMLAttributes<HTMLDivElement> {
+  disabled?: boolean;
+  icon?: React.ReactElement<any> | React.ReactElement<any>[];
+  isActive?: boolean;
+  onChange?: () => void;
+}
+
+const StyledTabs = styled(Tabs)`
+  flex: 0 auto;
+  position: sticky;
+  top: 0;
+  li {
+    &:after {
+      transition: none;
+    }
+  }
+`;
+
+const StyledContentWrapper = styled.div`
```

Comment:
Do we need this, or can we just use Flex if we just have one style?

Reply:
I just like the way it looks instead of an inline style, consistent with the rest of the wrappers too!